src/com/owncloud/android/authentication/AccountUtils.java

```diff
@@ -312,24 +314,25 @@ public static String trimWebdavSuffix(String url) {
     public static OwnCloudVersion getServerVersion(Account account) {
         OwnCloudVersion serverVersion = null;
         if (account != null) {
-            AccountManager accountMgr = AccountManager.get(MainApp.getAppContext());
-            String serverVersionStr = accountMgr.getUserData(account, Constants.KEY_OC_VERSION);
-            if (serverVersionStr != null) {
-                serverVersion = new OwnCloudVersion(serverVersionStr);
+            // capabilities are now the preferred source for version info
+            FileDataStorageManager fds = new FileDataStorageManager(
+                account,
+                MainApp.getAppContext().getContentResolver()
+            );
+            OCCapability capability = fds.getCapability(account.name);
+            if (capability != null) {
+                serverVersion = new OwnCloudVersion(capability.getVersionString());
+
+            } else {
+                // legacy: AccountManager as source of version info
+                AccountManager accountMgr = AccountManager.get(MainApp.getAppContext());
+                String serverVersionStr = accountMgr.getUserData(account, Constants.KEY_OC_VERSION);
+                if (serverVersionStr != null) {
+                    serverVersion = new OwnCloudVersion(serverVersionStr);
+                }
             }
         }
         return serverVersion;

```

Comment:
Added @Nullable to method to make it clearer to callers; they had it into account before, also.